addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.js

```diff
@@ -108,32 +77,14 @@ export class DiscussSidebarChannel extends Component {
             stateObserver: () => [this.floating?.isOpen],
         });
         this.floating = useDropdownState();
-        this.state = useState({
-            isSidebarCommandOpen: false,
-        });
-        this.popover = usePopover(DiscussSidebarChannelCommands, {
-            position: "right-start",
-            onClose: () => (this.state.isSidebarCommandOpen = false),
-            popoverClass:
-                "o-mail-DiscussSidebarChannelCommands min-width-0 p-2 bg-100 rounded-3 mx-1 border border-secondary",
-            arrow: false,
-        });
-        this.root = useRef("root");
-    }
-
-    get command() {
-        return {
-            onSelect: () => this.openSidebarCommands(),
-            title: this.title,
-            icon: "fa fa-ellipsis-h",
-        };
+        this.showingCommandState = useDropdownState();
```

Comment:
Maybe can reconcile `showingCommandState` into `floating`, so that exact same code with only `store.discuss.isSidebarCompact` that toggles between title on/off and whether actions are shown on click or on hover.

Reply:
Making `useHover()` conditional doesn't really makes it much simpler, so ok to keep it